mmedit/datasets/pipelines/coordinate.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import torch
+
+from ..registry import PIPELINES
+from .utils import make_coord
+
+
+@PIPELINES.register_module()
+class Coordinate:
```

Comment:
`Coordinate` is a verb in itself. To avoid confusion, we can use a different name like

Coordinate -> GenerateCoordinateAndCell